subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/Adb.kt

```diff
@@ -0,0 +1,16 @@
+package com.avito.runner.service.worker.device.adb
+
+class Adb {
+    val adb: String
+
+    init {
+        val androidHome: String = requireNotNull(System.getenv("ANDROID_HOME")) {
```

Comment:
Please consider using [AndroidSdk.kt](https://github.com/avito-tech/avito-android/blob/08d8bf607c449683b0b8e56bd43da386c1bae45c/subprojects/gradle/android/src/main/kotlin/com/avito/android/AndroidSdk.kt#L27) as a single place for sdk binaries access

Reply:
It is hard to reuse AndroidSdk.kt. I add a todo for further refactorings

Reply:
It seems that this knowledge should be passed to runner/service from outside.